### PR TITLE
Show number of selected options

### PR DESCRIPTION
**Pull request by rolljee**
Use case:

We got some little box where we put emails in this Multiselect.

Display need to be smaller so i added an option to display selectedOptions.length instead of all items joint by a coma.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-  "name": "@khanacademy/react-multi-select",
+  "name": "@rolljee/react-multi-select",
```
Comment (BrianGenisio) [inline package.json]: I can't merge this with this change.

File: package.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
   "version": "0.3.0",
   "description": "React Multi Select Component",
   "repository": {
     "type": "git",
-    "url": "https://github.com/khan/react-multi-select"
+    "url": "https://github.com/rolljee/react-multi-select"
```
Comment (BrianGenisio) [inline package.json]: Same with this.  Please revert in the PR.

File: src/index.js [hunk 3]
```diff
@@ -46,14 +47,19 @@ class MultiSelect extends Component<Props> {
...
-
-        const selectedOptions = selected
-            .map(s => options.find(o => o.value === s));
-
-        const selectedLabels = selectedOptions.map(s => s ? s.label : "");
-
-        return selectedLabels.join(", ");
+        const {options, selected, displayLabelsLength} = this.props;
+        if (displayLabelsLength) {
+            const selectedOptions = selected
```
Comment (BrianGenisio) [inline src/index.js]: Please pull this out of the `if()` block to avoid duplication.

File: src/stories/index.js [hunk 4]
```diff
@@ -163,7 +166,7 @@ class StatefulMultiSelect extends Component<SMSProps, SMSState> {
 
 function studentValueRenderer(selected, options) {
     if (selected.length === 0) {
-        return "Slect some students...";
+        return "Select some students...";
```
Comment (BrianGenisio) [inline src/stories/index.js]: Good catch!

**Comment by rolljee** [comment]
> @BrianGenisio @rolljee
> This can already be implemented using the `valueRenderer` prop. See [my answer](https://github.com/Khan/react-multi-select/issues/8#issuecomment-366449492).
> I think this PR should be closed.

I made this to have a simple and easy way to display
I don't really want to create a specific component just to achieve the same result using one props.

File: package.json [hunk 5]
```diff
@@ -1,10 +1,10 @@
 {
-  "name": "@khanacademy/react-multi-select",
-  "version": "0.3.0",
+  "name": "rolljee-react-multi-select",
```
[the remark this answers is not in the document]
Comment (rolljee) [reply package.json]: Alright i didn't knew about that thx

**Comment by HackedBeat** [comment]
@rolljee 
You do not need to create specific component to achieve this result. All you need to do is add this `valueRenderer` prop:
```js
<MultiSelect
  options={options}
  selected={selectedOptions}
  valueRenderer={(selected) => <span>{selected.length} items selected</span>}
/>
```
Is it difficult?

**Comment by rolljee** [comment]
Nope, didn't through i could do it that way !
If @BrianGenisio want to close, i'll close :)

**Comment by HackedBeat** [comment]
@rolljee Do not forget to handle the case when the `selectedOptions` array is empty or contains only one element, something like this:

```js
valueRenderer={(selected) => {
  if (!selected.length) return 'No items selected';
  return (
    <span>
      {selected.length > 1
        ? `${selected.length} items selected`
        : '1 item selected'
      }
    </span>
  );
}
```

**Comment by BrianGenisio** [comment]
@HackedBeat Ah, yes!  I totally forgot about that.  That's why the `valueRenderer` exists, and it doesn't make sense to have more than one way of doing things.  @rolljee assuming this works for you, we'll close this.  Thanks for sending the PR, though!  I love PRs (especially since I don't actively use this component myself anymore).

**Comment by rolljee** [comment]
Closing as a better option already exists.

Used the `valueRenderer` option instead.

Thx for pointing it @HackedBeat

**Pull request closed without merging**